ansible/ocs.yaml

```diff
@@ -0,0 +1,174 @@
+
+      - name: Restart {{ ocs_worker_domain }} VM
+        virt: 
+          name: "{{ ocs_worker_domain }}"
+          state: running
+
+      - name: Set {{ ocs_worker_domain }} VM live cpus
+        command: "virsh setvcpus {{ ocs_worker_domain }} {{ ocp_storage_vcpu }} --live"
+
+- hosts: localhost
+  gather_facts: false
+  become: true
+  user: root
+  vars_files: vars/default.yaml
+  roles:
+  - oc_local
+  tasks:
+  - name: Install OCS
+    when: ocs_enabled == true
+    block:
+    - name: Create OCS YAMLs working dir
+      file:
+        path: "{{ working_yamls_dir }}/ocs"
+        state: directory
+        mode: 0755
+
+    - name: Write OCS YAMLs to working dir
+      template:
+        src: ocs/{{ item }}.yaml.j2
+        dest: "{{ working_yamls_dir }}/ocs/{{ item }}.yaml"
+      with_items:
+        - local-storage-sub
+        - local-storage-volumes
+        - ocs-storage-cluster
+        - ocs-sub
+
+    - name: Label {{ ocs_worker_node }} as OCS-capable
+      shell: |
+          oc label node/{{ ocs_worker_node }} cluster.ocs.openshift.io/openshift-storage='' --overwrite=true;
+          oc label node/{{ ocs_worker_node }} topology.rook.io/rack=rack0 --overwrite=true
+      environment:
+        PATH: "{{ oc_env_path }}"
+        KUBECONFIG: "{{ kubeconfig }}"
+    
+    - name: Deploy local-storage subscription
+      shell: |
+        oc new-project local-storage
+        oc annotate project local-storage --overwrite openshift.io/node-selector=''
+        oc apply -f {{ working_yamls_dir }}/ocs/local-storage-sub.yaml
+      environment:
+        PATH: "{{ oc_env_path }}"
+        KUBECONFIG: "{{ kubeconfig }}"
+
+    - name: Wait for local-storage operator to be ready
+      shell: oc get pods -n local-storage
+      retries: 100
+      delay: 20
+      register: local_storage_operator_ready
+      until: (local_storage_operator_ready.stdout | regex_findall('local-storage-operator-.+-.+Running') | length) == 1
+      environment:
+        PATH: "{{ oc_env_path }}"
+        KUBECONFIG: "{{ kubeconfig }}"
+
+    - name: Create local-storage PVs
+      shell: |
+        oc apply -f {{ working_yamls_dir }}/ocs/local-storage-volumes.yaml
+      environment:
+        PATH: "{{ oc_env_path }}"
+        KUBECONFIG: "{{ kubeconfig }}"
+
+    - name: Deploy OCS subscription
+      shell: oc apply -f {{ working_yamls_dir }}/ocs/ocs-sub.yaml
+      environment:
+        PATH: "{{ oc_env_path }}"
+        KUBECONFIG: "{{ kubeconfig }}"
+
+    - name: Wait for OCS, rook-ceph and noobaa operators to be ready
+      shell: oc get pods -n openshift-storage
+      retries: 100
+      delay: 20
+      register: ocs_operators_ready
+      until: (ocs_operators_ready.stdout | regex_findall('ocs-operator-.+-.+Running') | length) == 1 and (ocs_operators_ready.stdout | regex_findall('rook-ceph-operator-.+-.+Running') | length) == 1 and (ocs_operators_ready.stdout | regex_findall('noobaa-operator-.+-.+Running') | length) == 1
```

Comment:
I opted to use `regex_findall` here (and elsewhere) instead of using `oc wait` in the shell because of what I mentioned in the last PR.  Namely, that there are too many transient errors that seem to pop up during the execution of `oc wait`.  It became very ugly to keep adding those erorrs to a `failed_when` conditional to try to keep the automation flow alive without having to re-run the playbook.